dockers/llm.rag.service/serverragllm_zendesk_csv_sql_local.py

```diff
@@ -130,7 +135,7 @@ def read_item(question: Union[str, None] = None):
 sql_search_db_and_model_path = os.getenv("SQL_SEARCH_DB_AND_MODEL_PATH", "/app/db/")
 
 app = setup(
-    file_path, relevant_docs, llm_server_url, model_id, max_tokens, model_temperature
+    file_path, relevant_docs, llm_server_url, model_id, max_tokens, model_temperature,  sql_search_db_and_model_path
 )
 
 
```

Comment:
If You only need this for local testing than maybe use the weaviate starting point and just exchange this line
```
search_type = question_router(question, sql_search_db_and_model_path)
```
with
```
search_type = SearchType.SQL
```
:)